Tests/Unit/EmogrifierTest.php

```diff
@@ -1519,4 +1519,45 @@ public function emogrifyAutomaticallyClosesUnclosedTag()
             $this->subject->emogrify()
         );
     }
+
+    /**
+     * @test
+     */
+    public function emogrifyDefaultReturn()
+    {
+        $this->subject->setHtml($this->html5DocumentType . '<html><body><p></body></html>');
```

Comment:
Could you please add the closing p tag in the new tests (as the tests are not about invalid XML)?